src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers } from 'redux';
+import reducerForCategory, { createActionForCategory } from './categories/categories';
+import reducer, { createActionForBookToAdd } from './books/books';
+
+const rootReducer = combineReducers({ books: reducer, categories: reducerForCategory });
+
+const store = createStore(rootReducer);
+
+(createActionForCategory());
+(createActionForBookToAdd());
```

Comment:
- [ ] Kindly consider adding `import { configureStore } from '@reduxjs/toolkit'` instead of createStore as we are told to use redux toolkit in this project. 
After importing configureStore, you need to make some changes in this file. you can learn how to do that in this [website](https://redux-toolkit.js.org/api/configureStore).